tests/test_components/test_autograd.py

```diff
@@ -635,57 +768,6 @@ def objective(*args):
         ag.grad(objective)(params0)
 
 
-def test_web_failure_handling(log_capture, monkeypatch, use_emulated_run, use_emulated_run_async):
```

Comment:
Because [this](https://github.com/flexcompute/tidy3d/pull/1824/files#diff-9f6ac729a84eb7e5b2ebe88ff4a4b52e63a0e21ec3e47d0553a9d0c1298b1a77L33-L45) was removed, but maybe the test needs to be modified instead?